packages/settings/README.md

```diff
@@ -0,0 +1,33 @@
+[![npm (scoped)](https://img.shields.io/npm/v/@ui5-language-assistant/settings.svg)](https://www.npmjs.com/package/@ui5-language-assistant/settings)
```

Comment:
Do we really need a whole settings **package**? (300+LOC) for a few wrappers around a `Map` and some default values?


What is the desired benefit?

Reply:
To be able to access it from different packages if necessary, and have the settings structure defined in a single place (except the client package.json, but I think we should add a test to ensure they're in sync).
The language-server obviously needs to access the settings because this is where they are updated. If we want validations or code completion to access it they shouldn't depend on the language-server.

Reply:
I added the test

Reply:
this could be resolved by separating the settings types and settings implementation. But its o.k. to start with a single settings package for now.